src/js/components/ConfigurationView.js

```diff
@@ -219,12 +305,17 @@ class ConfigurationView extends mixin(StoreMixin) {
       );
     }
 
+    let portsSection = this.getVIPPortDefinitionsSection(config);
+    if (!portsSection) {
+      portsSection = this.getPortDefinitionsSection(config);
+    }
```

Comment:
@orlandohohmeier could you take a peek at this? Not sure how to handle only showing one of them otherwise. I think to the user we should just show a "Ports Definition" because in the modal it shows as one grouping instead of two.